etc/contributing/run_postgres.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
What is this used for? Just debugging?

Reply:
It's nice to have postgres without minikube being spun up.  It also logs invalid queries/statements, which is more useful than the errors you get in Go from the driver.